app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/LoginActivity.java

```diff
@@ -43,9 +43,11 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             if (resultCode == RESULT_OK) {
                 // Successfully signed in
                 FirebaseUser user = FirebaseAuth.getInstance().getCurrentUser();
+                // Parse user information to the MainActivity (John is an example)
+                // TODO: Attach needed information to the intent
                 startActivity(
                         new Intent(this, MainActivity.class)
-                                .setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK)
+                                .setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK).putExtra("USER_NAME", "John")
```

Comment:
We might use a class instead of a string here, though it's not so important right now.